### PR TITLE
CGT-710: Disposal date post route and validation

### PR DESCRIPTION
**Pull request by MJCallahanPage**

### CODE REVIEW

File: app/common/Validation.scala [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package common
+
+import java.text.SimpleDateFormat
+
+object Validation {
```
Comment (DavidWoodward37) [inline app/common/Validation.scala]: Did you intend this class to be used for more than just date validation?

File: app/controllers/CalculationController.scala [hunk 2]
```diff
@@ -190,6 +190,16 @@ trait CalculationController extends FrontendController {
     }
   }
 
+  val submitDisposalDate = Action { implicit request =>
+    disposalDateForm.bindFromRequest.fold(
+      errors => BadRequest(calculation.disposalDate(errors)),
+      success => {
+        calcConnector.saveFormData("acquisitionValue", success)
```
Comment (DavidWoodward37) [inline app/controllers/CalculationController.scala]: Should this not be saved as disposalDate?

File: app/common/Validation.scala (hunk 1 above)
Comment (MJCallahanPage) [inline app/common/Validation.scala]: Potentially, I realised that the date validation would need to be in more than one place, which is why I tried to put this code in a 'shared/common' type of location. If we needed to add other functions to perform common validation they could go here. 

I'm not sure on what the 'best practice' is for things like this - may need some input from Mac.

File: app/controllers/CalculationController.scala (hunk 2 above)
Comment (MJCallahanPage) [inline app/controllers/CalculationController.scala]: Yep, I'll correct this now.

**Pull request merged**